main.nf

```diff
@@ -733,19 +790,14 @@ process bundle_summary {
         set val('tpm_filtered'), file('tpm_filtered_stats.csv') optional true into BUNDLE_SUMMARY_TPM
 
     """
-    celltype_markers_opt=
-    if [ -e '${params.cellTypeField}_markers.tsv' ]; then
-        celltype_markers_opt='--celltype-markers-file=${params.cellTypeField}_markers.tsv'
-    fi
-
     for matrix_type in filtered_normalised tpm_filtered; do
         if [ -d \${matrix_type}_dir ]; then
             makeMarkerStats.R \
                 --counts-dir=\${matrix_type}_dir \
                 --clusters-file=${clusters} \
-                --cluster-markers-dir=\$(pwd) \
-                --celltype-fields=${params.cellTypeField} \
-                --cellgroups-file=${cellMeta} \$celltype_markers_opt \
+                --cluster-markers-dir=cluster_markers \
```

Comment:
just out of curiosity, why are we using `=` here?

Reply:
Hmm, maybe that isn't necessary. But it works, so clearly allowed syntax.